cli/application/src/main/java/com/buschmais/jqassistant/commandline/task/RegisteredTask.java

```diff
@@ -136,12 +139,19 @@ public static List<Task> getTasks() {
         return tasks;
     }
 
-    public static List<String> getTaskNames() {
-        List<String> taskNames = new ArrayList<>();
+    public static Map<String, String> getTaskNamesAndDescriptions() {
+        Map<String, String> taskNameAndDescription = new HashMap<>();
         for (RegisteredTask registeredTask : values()) {
-            taskNames.add(CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.LOWER_HYPHEN, registeredTask.name()));
+            String description;
+            if(registeredTask.getTask().getClass().getAnnotation(Description.class) == null){
```

Comment:
`registeredTask.getTask().getClass().getAnnotation(Description.class)` is used here and below, please extract a variable .